packages/contractkit/src/kit.ts

```diff
@@ -64,7 +65,7 @@ export interface NetworkConfig {
 export interface KitOptions {
   gasInflationFactor: number
   feeCurrency: Address | null
-  from?: Address
+  from: Address | null
```

Comment:
wondering how hard it would be to prune this nulls

Reply:
i would keep `?: Address`.. or essentially `Address | undefined` and when you want to "unset" the address you call `setDefaultAccount(undefined)`